test/shared_functions.ts

```diff
@@ -151,6 +151,8 @@ export function evaluate(assert, expected, code, session = new Database()) {
   let {blocks, errors} = builder.buildDoc(parsed.results);
   if(expected.errors) {
     assert.true(parsed.errors.length > 0 || errors.length > 0, "This test is supposed to produce errors");
+  } else {
+    assert.true(parsed.errors.length == 0 && errors.length == 0, "This test is not supposed to produce errors");
```

Comment:
We just merged in changes to shared_functions.ts... this change should merge in fine, but it could cause a conflict. We'll see.